pom.xml

```diff
@@ -1281,6 +1281,11 @@
         <artifactId>args4j</artifactId>
```

Comment:
remove args4j since we don't use it ?

Reply:
not yet. we still have places other than PinotAdministrator that still uses args4j. there are also some project that extends pinot-tools which will be using args4j transitively.

Will completely remove args4j in the next couple of PRs once I migrate everything